Polyhedron/demo/Polyhedron/Scene.cpp

```diff
@@ -683,6 +683,23 @@ Scene::draw_aux(bool with_names, CGAL::Three::Viewer_interface* viewer)
         transparent_items.push_back(id);
     }
     renderScene(children, viewer, picked_item_IDs, with_names, -1, false, NULL);
+    if(with_names)
+    {
+      //here we get the selected point, before erasing the depth buffer. We store it 
+      //in a dynamic property as a QRectF because there is no QVector3D. If not found, we erase the property.
+      bool found = false;
```

Comment:
http://doc.qt.io/qt-5/qvector3d.html

Reply:
You meant that `QVector3D` is not compatible with `QVariant`.

Reply:
But even that is wrong:
http://doc.qt.io/qt-5/qvector3d.html#operator-QVariant

Reply:
I meant there is no toVector3D() function to convert a QVariant to a QVector3D.

Reply:
There is `QVector3D` is supported by `QVariant`, with `QVariant::fromValue` and `variant.value<QVector3D>()`. That API is valid for all types recognized by `QMetaType`.

Reply:
Ok, but I need a QList<QVariant> after all.